.ahoy.yml

```diff
@@ -78,14 +78,9 @@ commands:
 
   lint:
     usage: Lint code
-    cmd: docker-compose exec -T test ./vendor/bin/govcms-lint web/themes/custom
-
-  tests:
```

Comment:
removed `tests` as it is not used anywere. Per test type commands are used anyway.